src/main/cpp/DragonVision/DragonVision.cpp

```diff
@@ -25,6 +25,7 @@
 #include "DragonVision/DragonPhotonCam.h"
 #include "utils/FMSData.h"
 #include "DragonVision/DragonVisionStructLogger.h"
+#include "utils/logging/Logger.h"
```

Comment:
Remove debug logging

Reply:
done. the only logging left is in that test function that's never called